tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -496,3 +497,67 @@ def test_insights_tab_on_host_details_page(
                 assert recommendation['label'] == 'Moderate'
                 assert dnf_issue in recommendation['text']
                 assert len(insights_recommendations) == int(result['Recommendations'])
+
+
+@pytest.mark.run_in_one_thread
```

Comment:
why run_in_one_thread is needed here?

Reply:
Surely can remove that not a hurdle, just going for a precautious approach as I have multiple checkouts and the setting are tweaked on the Sat.